ccf_transaction_processor/generate_signing_keys/key_gen.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from infra.clients import CCFClient
+import argparse
+
+def run(args):
+    
+    host = args.host
+    port = 6006
+    cert = "./user1_cert.pem"
+    key = "./user1_privk.pem"
+    cafile="./networkcert.pem"
+    format = "json"
+
+    client = CCFClient(host, port, cert=cert, key=key, ca = cafile, format=format, prefix="users", description="none", \
+        version="2.0",connection_timeout=3, request_timeout=3)
+
+    r=client.rpc("generate_signing_key_for_read_payloads", dict())
```

Comment:
looking at the CCFClient definition, I think you're missing some error handling here, specifically a try/catch for exceptions.

Reply:
Thank you, so the line below this checks if the response has an error or not. basically, if the error comes from CCF iteslf, the code will report it. Yes, we do not catch an error while trying to submit the rpc in this script, though when we submit rpcs in PDO clients, I do check that as well. This script is basically a one time manual script, so I dont think it makes much difference. Thanks!